Golem/Golem.cs

```diff
@@ -181,12 +219,13 @@ public async Task<bool> Suspend()
 
         public Golem(string golemPath, string? dataDir, ILoggerFactory? loggerFactory = null)
         {
-            var prov_datadir = dataDir != null ? Path.Combine(dataDir, "provider") : null;
-            var yagna_datadir = dataDir != null ? Path.Combine(dataDir, "yagna") : null;
+            var prov_datadir = dataDir != null ? Path.Combine(dataDir, "provider") : "./provider";
+            var yagna_datadir = dataDir != null ? Path.Combine(dataDir, "yagna") : "./yagna";
```

Comment:
do we even create `provider` / `yagna` dirs outside of data dir or was it just `null` warning fix?

Reply:
null warning fix